CMSgov/onemac
# Responsive table style fixes

Story: https://qmacbis.atlassian.net/browse/OY2-21971

### Details

This PR is adding style updates to better suit mobile devices; adjusting things on the table page to scale better per the provided Figma: https://www.figma.com/file/DUtQSZ61dCDDKd4fAsz4TG/CMS-DS-2.5-Updates?node-id=0-1&t=h2dxfQq0k5sIZaxn-0

### Changes

- Scaling for page title bar
- Scaling for table tabs (SPA/WAIVER buttons)
- Scaling for search and filter accessors
- Wrapping for Type chip
- Wrapping for buttons
- Wrapping for flex-boxes

#### Screenshots
| Version | Before | After | HCD Mockup
| -------- | ------- | ------- | ------- |
| Phone | <img width="200" alt="Screen Shot 2023-03-27 at 1 43 44 PM" src="https://user-images.githubusercontent.com/4022304/228023640-b8af4c96-daef-4be4-99b8-347f038f1bed.png"> | <img width="200" alt="Screen Shot 2023-03-27 at 1 40 14 PM" src="https://user-images.githubusercontent.com/4022304/228023931-a3799771-e5b5-4ba5-b95c-47489926eec6.png"> | <img width="248" alt="Screen Shot 2023-03-27 at 1 58 21 PM" src="https://user-images.githubusercontent.com/4022304/228026888-fa45da14-c842-4b63-be43-ce8027e965e3.png">
| Tablet | <img width="400" alt="Screen Shot 2023-03-27 at 1 44 03 PM" src="https://user-images.githubusercontent.com/4022304/228023642-e8f606f0-7fcf-403a-b71e-283b9fa57329.png"> | <img width="400" alt="Screen Shot 2023-03-27 at 1 44 23 PM" src="https://user-images.githubusercontent.com/4022304/228023962-8a598806-ca0d-4319-b76d-63d0e076ecd5.png"> | <img width="422" alt="Screen Shot 2023-03-27 at 1 58 31 PM" src="https://user-images.githubusercontent.com/4022304/228026932-6583e7f2-ba02-4d53-ab7b-ca33733ab4b7.png">
| Laptop | <img width="1062" alt="Screen Shot 2023-03-27 at 1 55 07 PM" src="https://user-images.githubusercontent.com/4022304/228025999-6f0b7d50-f200-497c-88af-483093e48925.png"> | <img width="1060" alt="Screen Shot 2023-03-28 at 1 20 01 PM" src="https://user-images.githubusercontent.com/4022304/228318689-1c556476-dd55-45c8-92c9-459d49e17dff.png"> | <img width="613" alt="Screen Shot 2023-03-27 at 1 58 59 PM" src="https://user-images.githubusercontent.com/4022304/228026973-1409d999-918c-43f2-b1c9-0bc8a27d71e3.png">
| Desktop | <img width="1255" alt="Screen Shot 2023-03-27 at 1 55 34 PM" src="https://user-images.githubusercontent.com/4022304/228026130-4203a2cb-53d7-4b1e-a9ab-a89eb6611a3c.png"> | <img width="1260" alt="Screen Shot 2023-03-27 at 1 55 24 PM" src="https://user-images.githubusercontent.com/4022304/228026168-317a869e-3624-49ad-866a-0390b7f66410.png"> | <img width="585" alt="Screen Shot 2023-03-27 at 1 59 10 PM" src="https://user-images.githubusercontent.com/4022304/228027015-148c8ffb-9ef7-4b78-aa14-c5e52f3ddee0.png">

### Implementation Notes

Used inline media queries as I saw the same pattern used elsewhere in the SCSS file. Not a huge fan of how monolithic our styles are -- makes tangling things up pretty easy. Not a huge issue if we're ultimately moving to OneMAC 2.0 later, but a good topic to keep in mind for dev chats as we plan 2.0 architecture and developer patterns.

### Test Plan

1. Use the `Right-click > Inspect` method to open your browser dev tools window. 
2. Find the responsive design tool (the icon of a tablet and phone together) and click it so it's on (when it's the color in the photo)

<img width="229" alt="Screen Shot 2023-03-27 at 12 47 49 PM" src="https://user-images.githubusercontent.com/4022304/228009479-cbfed273-a7b5-4608-9598-4893b112c4b9.png">

3. Use the responsive tool to change window size in your browser and observe the scaling of the items in this PR's Changes section
